encrypted-config-value/src/main/java/com/palantir/config/crypto/algorithm/AesAlgorithm.java

```diff
@@ -40,7 +44,7 @@
     public static final String ALGORITHM_TYPE = "AES";
 
     private static final int GCM_AUTH_TAG_LENGTH = 128;
-    private static final int IV_LENGTH = 256 / Byte.SIZE;
+    private static final int IV_LENGTH = 96 / Byte.SIZE;
```

Comment:
Also, bigger question -- should we keep these as constants, but make them configurable but just have a default constructor/factory method that initializes with the default?